### PR TITLE
Loki: Update log context UI

### PR DESCRIPTION
**Pull request by ivanahuckova**
**What is this feature?**

This PR updates UI for Loki log context UI. The feature is experimental and behind the feature toggle. 

To be able to display executed query and to re-use the logic from `logContextProvider` we are passing `originalQuery` as param in the same way, as we are passing it to `getLogRowContext`.

The `text` is temporary and will be updated. 

image
image

### CODE REVIEW

File: public/app/plugins/datasource/loki/components/LokiContextUi.tsx [hunk 1]
```diff
@@ -136,108 +147,129 @@ export function LokiContextUi(props: LokiContextUiProps) {
   const parsedLabels = contextFilters.filter(({ fromParser }) => fromParser);
   const parsedLabelsEnabled = parsedLabels.filter(({ enabled }) => enabled);
 
+  const contextFilterToSelectFilter = useCallback((contextFilter: ContextFilter): SelectableValue<string> => {
+    return {
+      label: `${contextFilter.label}="${contextFilter.value}"`,
```
Comment (svennergr) [inline public/app/plugins/datasource/loki/components/LokiContextUi.tsx:152]: I think we need to escape the value here using `escapeLabelValueInSelector`:
image

File: public/app/plugins/datasource/loki/components/LokiContextUi.tsx [hunk 2]
```diff
@@ -136,108 +147,129 @@ export function LokiContextUi(props: LokiContextUiProps) {
...
-              });
-            }
-            if (actionMeta.action === 'remove-value') {
-              reportInteraction('grafana_explore_logs_loki_log_context_filtered', {
-                logRowUid: row.uid,
-                type: 'label',
-                action: 'remove',
-              });
+    <div className={styles.wrapper}>
+      <LoadingPlaceholder text="" className={`${styles.loadingPlaceholder} ${loading ? '' : styles.hidden}`} />
```
Comment (svennergr) [inline public/app/plugins/datasource/loki/components/LokiContextUi.tsx:159]: I think we can improve the display of the loading indicator. But that's definitely something for another PR.

**Review by svennergr — APPROVED** [review]
Just the addition of `escapeLabelValueInSelector` that needs to be done. The rest is looking awesome! 🚀

File: public/app/plugins/datasource/loki/components/LokiContextUi.tsx (hunk 1 above)
Comment (ivanahuckova) [reply public/app/plugins/datasource/loki/components/LokiContextUi.tsx]: Good catch! Fixed in https://github.com/grafana/grafana/pull/66730/commits/5af191b96aeec35e28b6aad44059a8e36791bc63

**Pull request merged**